src/main/java/com/funnyboyroks/worldeditselectionviewer/WorldeditSelectionViewer.java

```diff
@@ -51,7 +53,11 @@ public void onEnable() {
         ShapeRenderer defaultRenderer = new ShapeRenderer();
         defaultRenderer.setColor(Color.YELLOW);
         defaultRenderer.setForceShow(true);
+        defaultRenderer.setStepSize(0.2);
+        defaultRenderer.setOptimize(true);
         Bukkit.getScheduler().runTaskTimerAsynchronously(instance, () -> {
+            counter++;
+            counter &= 3;
```

Comment:
The Bukkit scheduler has a method for slowing down how frequently this body is called that would be more performant to use.  That is done by modifying the fourth argument to this function (currently `2`).

It may be also worth making this only slow down for certain players, so that players that don't experience performance issues can have the faster refresh rate, but that should probably be moved to its own PR.

Reply:
I would rather not have the lines blink unless the server owner wants it.  I do have have a config file in this plugin, but I'll add one soon and it can be put there.  For the time being, keep this at 2, so we don't affect servers.